eng/pipelines/common/global-build-job.yml

```diff
@@ -8,10 +8,12 @@ parameters:
   container: ''
   crossrootfsDir: ''
   variables: {}
-  timeoutInMinutes: ''
+  timeoutInMinutes: 120
```

Comment:
I am not sure this is part of the WIP, but you're changing the default for all legs, I guess?

Reply:
Yeah I think we should intentionally pass it down as 120 only for legs that run tests from runtime.yml as a jobParameter.

Reply:
I hit a 60 minute whole-job timeout during one of my runs, this was a quick hack to not run into that.

Reply:
https://dev.azure.com/dnceng/public/_build/results?buildId=688824&view=logs&jobId=108d2c4a-8a62-5a58-8dad-8e1042acc93c

Reply:
Yeah it makes sense for jobs that run tests, so we should just override the timeout in the job declaration in runtime.yml